src/backend/cuda/CMakeLists.txt

```diff
@@ -338,6 +338,7 @@ if(UNIX)
   if(CUDA_VERSION VERSION_GREATER 10.0)
     target_link_libraries(af_cuda_static_cuda_library
       PRIVATE
+        spdlog
```

Comment:
this is added in common interface target, dont we use common interface target with this static lib ?

. also it will fail when we looj for system/vcpkg provided spdlog target `spdlog::spdlog_header_only` -- perhaps I can handle this in my PR #3139